corehq/apps/callcenter/tasks.py

```diff
@@ -47,15 +47,10 @@ def calculate_indicators():
         indicator_set.get_data()
 
 
-def sync_web_user_usercases_if_applicable(user, domain, spawn_task):
+def sync_web_user_usercases_if_applicable(user, domain):
     domain_obj = Domain.get_by_name(domain)
-    if settings.UNIT_TESTING:
-        return
     if domain_obj.usercase_enabled and USH_WEB_USER_CASE_CREATION.enabled(domain):
```

Comment:
the FF check should come before the `domain_obj` is fetched / used. The reason being that we can avoid an unnecessary object fetch and also in tests the domain may not exist so it the variable could be `None`. For tests with the FF enabled they would always create the domain object.